### ISSUE TITLE
Old docker compose yml reference documentation link redirects to localhost

### ISSUE THREAD

**Issue by matthewfranglen** [docker/docker.github.io#2157]
### Problem description

I sometimes go to documentation by using my history. When I did that for the docker compose yml reference it suggested the https://docs.docker.com/v1.8/compose/yml/ URL. When I visited this I was redirected to http://localhost/compose/compose-file/

The most up to date URL is https://docs.docker.com/compose/compose-file/ so it looks like whatever service generates the redirect isn't resolving the hostname to use correctly.

### Problem location

- I saw a problem on the following URL: https://docs.docker.com/v1.8/compose/yml/

### Suggestions for a fix

```
➜ curl https://docs.docker.com/v1.8/compose/yml/
<!DOCTYPE html><html><head><link rel="canonical" href="http://localhost/compose/compose-file/"/><meta http-equiv="content-type" content="text/html; charset=utf-8" /><meta http-equiv="refresh" content="0;url=http://localhost/compose/compose-file/" /></head></html>
```

The redirect is being done using a meta refresh. I can't easily see what is generating that.

When I look at pages that have not been renamed (e.g. overview, gettingstarted) they work fine and link to the old documentation:

https://docs.docker.com/v1.8/compose/
https://docs.docker.com/v1.8/compose/gettingstarted/

The updated name even works:

https://docs.docker.com/v1.8/compose/compose-file/

I would suggest that this is a problem with whatever is handling renamed pages.

Check whatever generates this redirect page? It could be receiving proxied requests and taking the hostname from that.

Check your nginx configuration? It could generate a 301 redirect instead of using meta refresh.

Is this related to the version in the URL? The https://docs.docker.com/compose/yml/ URL without the version number goes to the "sorry 404" page so it may be something related to the specific /v1.8/ path.

**Comment by londoncalling** [comment]
@mstanleyjones @johndmulhausen  can we look at this together tomorrow?

**Comment by mstanleyjones** [comment]
@londoncalling I am fixing this in #2788. It is just a typo.

**Comment by mstanleyjones** [comment]
Closed via #2788
